### DISCUSSION TITLE
Slight weirdness in natural interface ordering

### DISCUSSION THREAD

**Discussion by candlerb** [netbox-community/netbox#6330] [General]
A minor point I noticed, that I'm not sure if can be easily improved or not.

* Create a virtual chassis between two switches: one has `g1/[1-26]` and the other has `g2/[1-26]`
* Make the first switch the master
* Create a LAG interface "lag1" on the master

When you view the list of interfaces, they come in this order:

```
g1/1
...
g1/26
lag1
g2/1
...
g2/26
```

I am guessing this is because all the interfaces on the box in position 1 (the master) are shown before all the interfaces on the box in position 2?  But there is no visual indication that one block of ports is separate from the next block of ports.  Hence it appears as if the lag interface is positioned out-of-order.  At least, I was expecting to scroll to the end to find "lag".

**Comment by jeremystretch** [comment]
By default, interfaces are ordered initially by parent device and then by name. We probably just need to override the ordering for the view to omit ordering by device. Want to open a bug report for this?

**Reply by candlerb** [reply]
Done, #6339

**Comment by jcralbino** [comment]
Can we provide in the drop down box actually a compose name , something like (device name > interface name )? 

It seems that the ordering is following this approach in the background.
